ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/-components/advisor-fields.tsx

```diff
@@ -0,0 +1,94 @@
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { UseFormReturn } from 'react-hook-form';
+
+import { MultiSelectField } from '@/components/form/multi-select-field';
+import {
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from '@/components/ui/accordion';
+import {
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+} from '@/components/ui/form';
+import { Switch } from '@/components/ui/switch';
+import { advisors } from '@/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/-types';
+import { CreateRunFormValues } from '@/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/create-run';
+
+type AdvisorFieldsProps = {
+  form: UseFormReturn<CreateRunFormValues>;
+};
+
+export const AdvisorFields = ({ form }: AdvisorFieldsProps) => {
+  return (
+    <div className='flex flex-row align-middle'>
+      <FormField
+        control={form.control}
+        name='jobConfigs.advisor.enabled'
+        render={({ field }) => (
+          <FormControl>
+            <Switch
+              className='my-4 mr-4 data-[state=checked]:bg-green-500'
+              checked={field.value}
+              onCheckedChange={field.onChange}
+            />
+          </FormControl>
+        )}
+      />
+      <AccordionItem value='advisor' className='flex-1'>
+        <AccordionTrigger>Advisor</AccordionTrigger>
+        <AccordionContent>
+          <FormField
+            control={form.control}
+            name='jobConfigs.advisor.skipExcluded'
+            render={({ field }) => (
+              <FormItem className='mb-4 flex flex-row items-center justify-between rounded-lg border p-4'>
+                <div className='space-y-0.5'>
+                  <FormLabel>Skip excluded</FormLabel>
+                  <FormDescription>
+                    A flag to control whether excluded scopes and paths should
+                    be skipped from the advisor.
+                  </FormDescription>
+                </div>
+                <FormControl>
+                  <Switch
+                    checked={field.value}
+                    onCheckedChange={field.onChange}
+                  />
+                </FormControl>
+              </FormItem>
+            )}
+          />
+          <MultiSelectField
+            form={form}
+            name='jobConfigs.advisor.advisors'
+            label='Enabled advisors'
+            description={<>Select the advisors enabled for this ORT Run.</>}
```

Comment:
Just an info-nit, string can be directly passed as `ReactNode`, so no need to wrap them in Fragment.

Reply:
Yeah I know, but I would like to keep it as it is, to hint the user of the component that this is a ReactNode, so descriptions can include JSX.